config.json

```diff
@@ -1,4 +1,12 @@
 {
+    "account": {
+        "completeLifetime": {
+            "months": 6
+        },
+        "initiateLifetime": {
+            "hours": 1
+        }
+    },
```

Comment:
The account.completeLifetime, account.initiateLifetime and htop settings should all be documented.